infra/gp-cluster-monitoring-config/helper/update_alertmanager_config_in_cluster.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+echo "Es wird ein Update auf das SealedSecret gemacht."
+echo "Das SealedSecret wird dann widerum zum secret alertmanager-main."
```

Comment:
wiederum*
Eventuell Zusatzausgabe, dass ein "alertmanager.yml" in dem directory vorhanden sein muss.